### ISSUE TITLE
Error importing converted vcf

### ISSUE THREAD

**Issue by ratibida** [AgResearch/KGD#35]
I successfully converted my vcf file to the tassel format with the supplied python helper script (thanks!), but there seems to be an issue when running Gfull <-calcG(). 

Error in alleles[iind, ] <- matrix(as.numeric(unlist(strsplit(genosin[[iind +  : 
  number of items to replace is not a multiple of replacement length 

Would you have any solutions for this? 

Thanks!

**Comment by doddsk** [comment]
Thanks for the alert on this issue.
The error occurs when reading the (tassel format) data file (not with calcG). Are you able to share a few lines of your data file to help us sort this issue out?

**Comment by ratibida** [comment]
You are correct - the error does appear after reading the tassel format data file source("GBS-Chip-Gmatrix.R") not when running calcG(). I've attached a portion of my data file!

[f1v2.vcf.ra.txt](https://github.com/AgResearch/KGD/files/3698767/f1v2.vcf.ra.txt)

**Comment by r-ashby** [comment]
Hi, it looks like there is an issue with your RA file. Are you able to please share your VCF file and the pipeline you used to generate the VCF file?

**Comment by ratibida** [comment]
I used the GBS_SNP_CROP pipeline and attached my VCF file below. Thanks!
[f1.zip](https://github.com/AgResearch/KGD/files/3714052/f1.zip)

**Comment by ratibida** [comment]
Apologies, I mistakenly closed the issue...

**Comment by ratibida** [comment]
I used the [GBS_SNP_CROP](https://github.com/halelab/GBS-SNP-CROP) pipeline and attached my vcf file.
[f1.zip](https://github.com/AgResearch/KGD/files/3714069/f1.zip)

**Comment by doddsk** [comment]
Thanks for sending the vcf. We had not realised that the AD field was being used in (at least) 2 different ways. The script was written to expect a pair of read depths (ref,alt) associated with each genotype for this field. (This is how GATK and samtools use this label). We are looking at how much effort might be required to convert this type of vcf.

**Comment by rbrauning** [comment]
Hi, I've tried reproducing your problem (malformed AD field) in v3.0, v.4.0, v4.1 of GBS-SNP-CROP. The AD field looks fine in all versions and properly reports pairs of read depths (ref, alt), not like in your VCF file. What version of GBS-SNP-CROP did you use? I'm recommending script 8 of v4.0 to produce VCF files as the other versions have some other not AD related problems.
Could it be that you've further processed the output of GBS-SNP-CROP with some other tools? I've noticed some bcftools information in your header.
v4.0 VCF file or GSC.GenoMatrix.txt together with the barcode file would have all the information you need for KGD.
